src/chrome/content/rules/Pushsquare.com.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	Nonfunctional hosts in *.pushsquare.com:
+
+# All sites remain on http with this ruleset
+-->
+<ruleset name="Pushsquare.com">
+
+    <target host="pushsquare.com" />
+    <target host="www.pushsquare.com" />
+    <target host="images.pushsquare.com" />
+    <target host="static.pushsquare.com" />
+
```

Comment:
Add `<securecookie host=".+" name=".+" />` here.

Reply:
Thanks for the help mate